.github/workflows/unit-test.yml

```diff
@@ -38,5 +45,24 @@ jobs:
         with:
           go-version: "1.21"
 
+      - name: Setup jq
+        uses: dcarbone/install-jq-action@v2.0.2
+        with:
+          version: "${{ inputs.version }}"
+
+      - name: Check jq
+        run: |
+          which jq
+          jq --version
+
       - name: run unit tests
-        run: GOPATH=$(go env GOPATH) make -j $(nproc) test
+        run: |
+          export ROOK_UNIT_JQ_PATH="$(which jq)"
+          GOPATH=$(go env GOPATH) make -j $(nproc) test | tee output.txt
+          if grep "skipping mds liveness probe script unit tests because jq binary location is not known" output.txt
```

Comment:
Why would jq not be found? We know it's there because of the step added above. Can we remove all these checks on line 62-68 and just keep this step focused on running the unit tests?

Reply:
that would be great if we kept it just for running unit tests,

But at the same time checking the jq makes sense as per the unit test file maybe I will separate this out